colossalai/fx/profiler/constants.py

```diff
@@ -0,0 +1,32 @@
+import torch
```

Comment:
The file `colossalai/fx/profiler/constant.py` is deleted and then re-created? You should change the file directly instead of deleting it to minimize git change.

Reply:
I split this file into two, so git cannot identify this as a filename change.